tests/qtest/libqos/virtio-video.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * libqos virtio-video definitions
+ *
+ * Copyright (c) 2023 Red Hat Inc.
+ *
+ * Authors:
+ *  Albert Esteve <aesteve@redhat.com>
+ *   (based on virtio-gpio.h)
+ * 
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ */
+
+#ifndef TESTS_LIBQOS_VIRTIO_VIDEO_H
+#define TESTS_LIBQOS_VIRTIO_VIDEO_H
+
+#include "qgraph.h"
+#include "virtio.h"
+#include "virtio-pci.h"
+
+typedef struct QVhostUserVideo QVhostUserVideo;
```

Comment:
Why don't you use this (shorter) style to unite `typedef` and `struct` declaration? AFAIR it's possible.
```
typedef struct QVhostUserVideo {
    QVirtioDevice *vdev;
    QVirtQueue **queues;
} QVhostUserVideo;
```

Reply:
It is possible, yes. This is just copying the code style of other headers in the same folder. I'll keep it as it is for consistency, although I would probably do it as you suggested.